st2common/st2common/bootstrap/runnersregistrar.py

```diff
@@ -400,6 +400,30 @@
 
     # Experimental runners below
     {
+        'name': 'announcement',
+        'aliases': [],
+        'description': 'A runner for emitting an announcement event on the stream.',
+        'enabled': True,
+        'runner_parameters': {
+            'experimental': {
```

Comment:
@lakshmi-kannan was worried users may start using it before it is ready. I myself agree that it might not be such a big problem based on our previous experience, but decided to do it just in case.